src/commands/update.js

```diff
@@ -56,8 +56,12 @@ export default class Update extends Command {
     debug('log chop')
     await this.logChop()
     debug('autocomplete')
-    let ac = new AutocompleteScripter(this)
-    await ac.generateCommandsCache()
+    if (this.config.windows) {
+      debug('skipping autocomplete on windows')
```

Comment:
rather than this I think it makes more sense just to move the `debug('autocomplete')` into the autocomplete conditional